docs/source/project_documentation/dev_guide/debugging_aac.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+title: Debugging AaC
+parent: Developer's Guide to AaC
+nav_order: 1
+---
```

Comment:
This header information can go away, its not necessary with the Sphinx builds anymore.